superset/assets/package.json

```diff
@@ -99,7 +99,7 @@
     "jquery": "3.1.1",
     "json-bigint": "^0.3.0",
     "lodash": "^4.17.11",
-    "mapbox-gl": "^0.53.0",
+    "mapbox-gl": "^0.53.x",
```

Comment:
I think it's preferred to just pin directly to `0.53.1`

Reply:
Done